nemo_rl/models/policy/utils.py

```
@@ -381,258 +381,226 @@ def rebuild_cuda_tensor_from_ipc(
+        return
+
+    num_dtypes = len(gathered[0])
+    assert num_dtypes > 0
+    for i in range(num_dtypes):
+        body = {
+            "serialized_named_tensors": [g[i] for g in gathered],
+            "load_format": "flattened_bucket",
+            "flush_cache": False,
+            "weight_version": str(weight_version),
+        }
+        response = requests.post(f"{engine_url}/update_weights_from_tensor", json=body)
```

Ruff (0.15.12)

[error] Probable use of `requests` call without timeout

(S113)